website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,195 @@
+
+## Font families
+
+In code, Helios uses default system fonts because they are stable, support internationalization, and improve performance by reducing resource load—key for large applications at HashiCorp.
+
+In Figma, our design tooling and libraries use SF Pro font family exclusively and we recommend using this font stack for consistency.
+
+While text styling varies slightly across operating systems, users will have a consistent and expected experience within each OS.
+
+### Sans-serif
+
+#### Mac + iOS sans-serif font
+
+Large text (20px and above): SF Pro Display
+
+![Sans-serif display on Mac](/assets/foundations/typography/typography-sans-serif-display-mac.png)
+
+Small text (19px and lower): SF Pro Text
+
+![Sans-serif text on Mac](/assets/foundations/typography/typography-sans-serif-mac.png)
+
+#### Windows sans-serif font
+
+Large text (20px and above): Segoe UI Display
+
+![Sans-serif display on Mac](/assets/foundations/typography/typography-sans-serif-display-windows.png)
+
+Small text (19px and lower): Segoe UI Text
+
+![Sans-serif text on Mac](/assets/foundations/typography/typography-sans-serif-windows.png)
+
+#### Linux sans-serif font
+
+Linux sans-serif system font dependant on the Linux distribution
+
+![Sans-serif font for Linux.](/assets/foundations/typography/typography-sans-serif-linux.png)
+
+### Monospace
+
+#### Mac + iOS monospace font
+
+Safari 13+: SF Mono
+
+![Monospace for Safari 13+ on Mac](/assets/foundations/typography/typography-monospace-safari-13-mac.png)
+
+All else: Menlo
+
+![Monospace for all other versions on Mac](/assets/foundations/typography/typography-monospace-menlo-mac.png)
+
+#### Windows
+
+Consolas
+
+![Monospace on Mac](/assets/foundations/typography/typography-monospace-menlo-mac.png)
+
+#### Linux
+
+Linux monospace system font dependant on the Linux distribution
+
+![Monospace on Linux](/assets/foundations/typography/typography-monospace-linux.png)
+
+## Font weights
+
+In addition to font size, font weight helps create different levels of emphasis throughout the UI. Font weights include:
+
+- 400 (regular)
+- 500 (medium)
+- 600 (semi bold)
+- 700 (bold)
+
+Within these ranges, bold is used only for display and some code text styles, while regular is associated with body text styles which may include code.
+
+## Text styles
+
+### Display
+
+Display styles should be used for headings or visual emphasis (like remaining balances). Unless used for visual emphasis, if the display text looks like a page heading, it should use an [HTML heading tag](https://www.w3.org/WAI/WCAG22/Understanding/section-headings.html).
+
+Display text styles are available in the following sizes and weights:
+
+<Hds::Text::Display @size="500" @weight="bold">Display 500 (30px) - Bold</Hds::Text::Display>
+<br/>
```

Comment:
Because this is presented as a list, I really think it should have the list semantics. The benefit is that users can know how many items there are.

Ideally, you use the semantic list HTML elements and add styles to remove the bullet points:
```handlebars
<ul class='doc-typography-demo-display-list'>
  <Hds::Text::Display @tag="li" @size="500" @weight="bold">Display 500 (30px) - Bold</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display @tag="li" @size="400" @weight="bold">Display 400 (24px) - Bold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display @tag="li" @size="400" @weight="semibold">Display 400 (24px) - Semibold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display @tag="li" @size="400" @weight="medium">Display 400 (24px) - Medium</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display @tag="li" @size="300" @weight="bold">Display 300 (18px) - Bold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display @tag="li" @size="300" @weight="semibold">Display 300 (18px) - Semibold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display @tag="li" @size="300" @weight="medium">Display 300 (18px) - Medium</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display @tag="li" @size="200" @weight="semibold">Display 200 (16px) - Semibold</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display @tag="li" @size="100" @weight="medium">Display 100 (13px) - Medium</Hds::Text::Display>
</ul>
```

or you can manually set the roles
```handlebars
<div role='list'>
  <Hds::Text::Display role='listitem' @size="500" @weight="bold">Display 500 (30px) - Bold</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display role='listitem' @size="400" @weight="bold">Display 400 (24px) - Bold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display role='listitem' @size="400" @weight="semibold">Display 400 (24px) - Semibold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display  role='listitem' @size="400" @weight="medium">Display 400 (24px) - Medium</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display role='listitem' @size="300" @weight="bold">Display 300 (18px) - Bold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display role='listitem' @size="300" @weight="semibold">Display 300 (18px) - Semibold</Hds::Text::Display>
  <br/>
  <Hds::Text::Display role='listitem' @size="300" @weight="medium">Display 300 (18px) - Medium</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display role='listitem' @size="200" @weight="semibold">Display 200 (16px) - Semibold</Hds::Text::Display>
  <br/>
  <br/>
  <Hds::Text::Display role='listitem' @size="100" @weight="medium">Display 100 (13px) - Medium</Hds::Text::Display>
</div>
```

Reply:
Same for body and code.

Reply:
In this case, following @shleewhite suggeston, consider using plain HTML (`<ul>` + `<li>`) with CSS typographic class names (it would make the code much easier to interpret). See how it's done here: https://helios.hashicorp.design/foundations/typography?tab=code#style-and-weight (eg. `hds-typography-display-500 hds-font-weight-bold` for "Display/500/bold"